hs_sitemap/sitemaps.py

```diff
@@ -11,3 +13,17 @@ def items(self):
 
```

Comment:
Should we be including pages for 'groups' and 'communities' in `items `for `PagesSitemap`? This is a separate issue though.

Reply:
good idea, I modified to include groups and communities, thanks!